lib/fastifySession.js

```diff
@@ -192,19 +191,14 @@ function session (fastify, options, next) {
 
   function isConnectionSecure (request) {
     return (
-      request.raw.socket?.encrypted === true ||
-    request.headers['x-forwarded-proto'] === 'https'
+      request.raw.socket?.encrypted === true || request.headers['x-forwarded-proto'] === 'https'
     )
   }
 
-  function shouldSaveSession (request, cookieOpts, saveUninitialized) {
-    if (!saveUninitialized && !request.session.isModified()) {
-      return false
-    }
-    if (cookieOpts.secure !== true || cookieOpts.secure === 'auto') {
-      return true
-    }
-    return isConnectionSecure(request)
+  function getShouldSaveSession (request, cookieId, saveUninitializedSession, rollingSessions) {
```

Comment:
getShouldSaveSession? why two verbs? shouldSaveSession seems still better.

the variable you instantiate in line 142 can be called saveSession or shouldSave

Reply:
Hey man you're the one that said:

> Please dont shorten the variable name.

😛 

But point taken, will do